flake8_print.py

```diff
@@ -1,54 +1,38 @@
-import ast
-import tokenize
+"""Extension for flake8 that finds usage of print."""
+import re
 
-from sys import stdin
-
-__version__ = '1.6.1'
+__version__ = '1.7.0'
```

Comment:
As I've now changed the T001 to be either 'print statement found.' or 'print function found.' , and nearly everything else has been modified except the test cases, if you are using http://semver.org/ , this should really be 2.0.0